.storybook/main.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+    stories: ['../src/**/*.stories.tsx'],
+    addons: ['@storybook/addon-knobs',
+    'storybook-readme',
+    '@storybook/addon-a11y',
+    '@storybook/addon-viewport'
+],
+  };
```

Comment:
Did you run the linter? It'd likely automatically add the empty lines to the end of these files.

Reply:
Ah I think I forgot to run linter after the last change. Running it now !